README.md

```diff
@@ -436,6 +436,13 @@ Configuration settings that have been copied from vim. Vim settings are loaded i
 * What key should `<leader>` map to in key remappings?
 * Type: string (Default: `\`)
 
+#### `"vim.whichwrap"`
+
+* Controls wrapping at beginning and end of line.
+* Comma-separated set of keys that should wrap to next/previous line. Arrow keys are represented by `[` and `]` in insert mode, `<` and `>` in normal and visual mode.
```

Comment:
I chose to emulate whichwrap's (odd) syntax rather than lean on json to make a more understandable configuration option. Could go either way on that.

Reply:
I think this is a fine call to me, it is documented here, and is the vim way so why not